forge-gui/res/cardsfolder/upcoming/vault_112_sadistic_simulation.txt

```diff
@@ -0,0 +1,14 @@
+Name:Vault 112: Sadistic Simulation
+ManaCost:2 U R
+Types:Enchantment Saga
+K:Chapter:3:DBTap,DBTap,DBChooseNumber
+SVar:DBTap:DB$ Tap | ValidTgts$ Creature | SubAbility$ DBCounter | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target creature | SpellDescription$ Tap up to one target creature and put a stun counter on it. You get {E}{E} (two energy counters).
+SVar:DBCounter:DB$ PutCounter | Defined$ Targeted | CounterType$ Stun | CounterNum$ 1 | SubAbility$ DBEnergy
+SVar:DBEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 2
+SVar:DBChooseNumber:DB$ ChooseNumber | Max$ Max | ListTitle$ Pay Energy for ability| SubAbility$ DBExile | SpellDescription$ Pay any amount of {E}. If you paid one or more {E} this way, shuffle your library, then exile that many cards from the top. You may play one of those cards without paying its mana cost.
+SVar:DBExile:DB$ Dig | UnlessCost$ PayEnergy<X> | UnlessPayer$ You | UnlessSwitched$ True | Defined$ You | DigNum$ X | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Valid$ Card.IsRemembered | ValidSA$ Spell | ValidZone$ Exile | WithoutManaCost$ True | Controller$ You | Optional$ True | Amount$ 1
```

Comment:
wrong valid

Reply:
Hmm i guess you would also be able to play lands ?

Reply:
yes